programs/lfx-mentorship/2025/01-Mar-May/project_ideas.md

```diff
@@ -19,4 +19,39 @@
 
 ## Proposed Project ideas
 
+### Meshery
 
+#### Meshery Model Support for kro ResourceGraphDefinitions (RGDs)
+
+- Description: Enhance Meshery's existing orchestration capabilities to include support for kro ResourceGraphDefinitions (RGDs) as first-class [Meshery Models](https://docs.meshery.io/concepts/logical/models). This involves enabling Meshery to manage and orchestrate RGDs, similar to how it handles other Kubernetes resources.  The project will also include generating support for ResourceGraphDefinition in Meshery's Model generator.
+
+- Expected Outcome:  Meshery will be able to orchestrate and manage kro RGDs.  This includes the ability to deploy, configure, and manage the lifecycle of RGDs through Meshery.  The Meshery Model generator will be updated to automatically generate models for kro RGDs, simplifying their integration and management within Meshery. This will be an officially supported feature of Meshery.
+- Recommended Skills: Golang, Cuelang, Well-written and well-spoken English, Kubernetes, DevOps
+- Mentor(s): Lee Calcote (@leecalcote, leecalcote@gmail.com), Mia Grenell (@miacycle, mia.grenell2337@gmail.com)
+- Upstream Issue: https://github.com/meshery/meshery/issues/13520
+
+#### Hands-on tutorials using Meshery Playground
+
+- Description: Learning paths with hands-on labs are a crucial resource for DevOps engineers and cloud-native practitioners. The Meshery Playground provides a live cluster environment, making it an ideal platform for learning every kind of cloud and cloud native technology. Meshery Docs is in need of comprehensive tutorials and scenarios covering common infrastructure management use cases.
+
+Your mission in this internship is to create and publish a series of hands-on tutorials using Meshery Playground. Each tutorial will include step-by-step guides, live demonstrations, and interactive labs using the Playground allowing learners to apply their knowledge directly without the hassle of any configuration.These tutorials will be reviewed by various project maintainers and then published in [guides/tutorials](https://docs.meshery.io/guides/tutorials).
+
+- Expected Outcome:
+  - 10+ new tutorials published in Meshery Docs.
+  - Each tutorial should be interactive, guiding users through infrastructure management scenarios.
+  - Tutorials should vary in complexity, catering to beginners and advanced learners
+- Recommended Skills: written English, Markdown, Kubernetes, DevOps, and hands-on experience with cloud-native tools.
+- Mentor(s): Lee Calcote (@leecalcote, leecalcote@gmail.com), James Horti (@hortison, james.horton2337@gmail.com)
+- Upstream Issue: https://github.com/meshery/meshery/issues/13521
```

Comment:
This looks like it might be a good fit for Google Season of Docs--do you think we may be able to shift it there? It's not a mentorship program, they hire experienced TechWriters so may not need as much mentor oversight.